install/connections.csv

```diff
@@ -741,6 +741,7 @@ RIZ,10,Raoul Island Met Station,,0,2009-05-18T00:01:00Z,9999-01-01T00:00:00Z
 RIZ,20,Raoul Island Met Station,,3,2009-05-18T01:00:00Z,2016-09-17T13:00:00Z
 RIZ,20,Raoul Island Met Station,Strong Motion,,2016-09-17T13:00:01Z,2023-01-28T21:16:00Z
 RKIA,10,Rakaia,Short Period,,1995-11-09T00:00:01Z,1996-04-18T23:59:59Z
+RLGLK,04,Raoul Green Lake,,,2023-01-30T02:59:59Z,9999-01-01T00:00:00Z
```

Comment:
since this site is co-located with GLKZ, I suggest to use the same Name
can you please change `Raoul Green Lake` to `Raoul Island Green Lake` ?

GLKZ entry for this is:
`GLKZ,10,Raoul Island Green Lake,,0,2009-05-19T00:00:00Z,9999-01-01T00:00:00Z`